gprofiler/usage_loggers.py

```diff
@@ -11,7 +11,17 @@
 
 import psutil
 
-CGROUPFS_ROOT = "/sys/fs/cgroup"  # TODO extract from /proc/mounts, this may change
+from granulate_utils.linux.cgroups.cgroup import find_v1_hierarchies, find_v2_hierarchy
+
+
+# TODO(Creatone): Move it to granulate-utils.
+def _obtain_cgroup_controller_path(cgroup: str, controller: str) -> str:
+    cgroup_v1_hierarchies = find_v1_hierarchies()
+    if len(cgroup_v1_hierarchies) != 1:
+        assert controller in cgroup_v1_hierarchies
+        return f"{cgroup_v1_hierarchies[controller]}{cgroup}"
+    else:
+        return f"{find_v2_hierarchy()}/{controller}{cgroup}"
```

Comment:
I don't think this will work now - because cgroups v2 files are different. For example - there is no `cpuacct.usage` file, but `cpu.stat` file which contains a `usage_usec` field.

I created a [ticket](https://github.com/Granulate/gprofiler/issues/567) for cgroups v2 support. Until that's done, I suggest you raise an exception here if v2 is in use.

Also I see that granulate-utils does this check: `len(get_cgroups(os.getpid())) == 1` (in `_verify_preconditions`). It's subtly different from what you did here (checks the controllers available for this processes instead of the mounted controllers). I think it'll produce the same results, but let's be consistent.